src/wdio/services/TerraService.js

```diff
@@ -41,6 +42,9 @@ export default class TerraService {
       /* `describeViewports` provides a custom Mocha `describe` block for looping test viewports. */
       describeViewports: viewportHelpers.describeViewports,
 
+      /* `hideInputCaret` hides the blinking input caret that appears in editable text areas. */
+      hideInputCaret,
```

Comment:
I worry people will not understand this need to be executed in an `before`, `beforeEach` or `it` block since `Terra.viewports` and `Terra.describeViewports` should not be used there.

Reply:
Perhaps it could be moved to be `Terra.should.hideInputCaret` along with the other functions that need to be in a `before`, `beforeEach` or `it` block.

Reply:
Nevermind, `should` is gone. Not sure then.

Reply:
@Blackop778 Thought about this for a while and talked with @mjhenkes. We both agree regardless of this naming here, there is potential for misuse if documentation is not read. I think this is fine !